core/net/ipv6/uip-ds6-route.h

```diff
@@ -162,7 +196,7 @@ uip_ipaddr_t *uip_ds6_route_nexthop(uip_ds6_route_t *);
 int uip_ds6_route_num_routes(void);
 uip_ds6_route_t *uip_ds6_route_head(void);
 uip_ds6_route_t *uip_ds6_route_next(uip_ds6_route_t *);
-
+int uip_ds6_route_is_nexthop(const uip_lladdr_t *lladdr);
```

Comment:
Any chance we can use ipaddr instead of lladdr as a parameter here? Would really increase consistency of the routing module interface. All other functions are ipaddr-based, including the similarly-named `uip_ds6_route_rm_by_nexthop`.